kanidm_tools/src/cli/session.rs

```diff
@@ -453,8 +460,10 @@ impl SessionOpt {
                     error!("Error persisting authentication token store");
                     std::process::exit(1);
                 };
-
-                println!("Removed {} sessions", start_len - end_len);
+                match dopt.prompt {
```

Comment:
same as above, I could remove this and use  kanidm_client libs

Reply:
Yeah, I think this is starting to look in favour of seperate tool then since the client libs are the contact point, and that avoids the need to feature gate.

Reply:
IE perhaps start kanidm_tui in parallel to kanidm_tools?

Reply:
sounds good to me, I would create kanidm_tui folder and remove all dependency to cli libs,